create-gitops-pull-request.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# Please Use Google Shell Style: https://google.github.io/styleguide/shell.xml
+
+# ---- Start unofficial bash strict mode boilerplate
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -o errexit  # always exit on error
+set -o errtrace # trap errors in functions as well
+set -o pipefail # don't ignore exit codes when piping output
+set -o posix    # more strict failures in subshells
+# set -x          # enable debugging
+
+IFS=$'\n\t'
+# ---- End unofficial bash strict mode boilerplate
+
+if [ -z "$SERVICE" ]; then
+  echo "Please set the SERVICE environment variable i.e hydra, keto etc"
+  exit 1
+fi
+
+if [ -z "$ENVIRONMENT" ]; then
+  echo "Please set the ENVIRONMENT environment variable i.e dev, staging etc"
+  exit 1
+fi
+
+
+if [ -z "$DOCKER_REPOSITORY" ]; then
+  echo "Please set the DOCKER_REPOSITORY environment variable i.e dev, staging etc"
+  exit 1
+fi
+
+if [ -z "$REACTION_GITOPS_GH_TOKEN" ]; then
+  echo "Please set the REACTION_GITOPS_GH_TOKEN environment variable."
+  exit 1
+fi
+
+if [ -z "$GH_USERNAME" ]; then
+  echo "Please set the GH_USERNAME environment variable."
+  exit 1
+fi
+
+if [ -z "$GH_EMAIL" ]; then
+  echo "Please set the GH_EMAIL environment variable."
+  exit 1
+fi
+
+if [ -z "$REACTION_GITOPS_REVIEWERS" ]; then
+  echo "Please set the REACTION_GITOPS_REVIEWERS environment variable."
+  exit 1
+fi
+
```

Comment:
Optional change but I'd prefer this were a loop that would print out all missing env vars in one go then exit so you don't fix one, fail one 2nd one, fix 2nd, fail on 3rd, etc. Something like this:

```sh
error=""
for VAR in USER USERNO1 USERNO2; do
  if [[ -z "${!VAR}" ]]; then
    error="${error}Missing required environment variable: ${VAR}\n"
  fi
done
if [[ -n "${error}" ]]; then
  echo -e "${error}" 1>&2
  exit 1
fi
```

Reply:
updated